stdlib/public/core/IntegerParsing.swift

```diff
@@ -77,24 +77,24 @@ where Rest.Element: UnsignedInteger {
 internal func _parseASCII<
   CodeUnits: IteratorProtocol, Result: FixedWidthInteger
 >(
-  codeUnits: inout CodeUnits, radix: Result
+  codeUnits: inout CodeUnits, radix: Int
 ) -> Result?
 where CodeUnits.Element: UnsignedInteger {
   let c0_ = codeUnits.next()
   guard _fastPath(c0_ != nil), let c0 = c0_ else { return nil }
   if _fastPath(c0 != _ascii16("+") && c0 != _ascii16("-")) {
     return _parseUnsignedASCII(
-      first: c0, rest: &codeUnits, radix: radix, positive: true)
+      first: c0, rest: &codeUnits, radix: radix, positive: true) as? Result
```

Comment:
This is a clear indication that this is not a full solution; `_parseUnsignedASCII` should still be returning a `Result`, so this cast shouldn't be necessary.

Reply:
@theblixguy Looks to me like you'll need to propagate the `radix: Result` -> `radix: Int` change throughout all the internal methods in this file.